protocol-list.md

```diff
@@ -41,3 +41,10 @@ publications.  The protocol is available in
 [Fujii et al, 2014](https://www.nature.com/articles/nprot.2014.107).
 There is also a [JOVE article](https://www.jove.com/video/55282).
 
+## Spencer et al,JOVE,2013
```

Comment:
You should add spaces after the commas and change JOVE to JoVE (the 'o' is lower case).

Reply:
Also: add a blank line after the section title (the one starting with ##).